### PR TITLE
Add recipe for cdsdashboards

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/cdsdashboards/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+build:
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+      script: "python -m pip install . --no-deps --ignore-installed -vv"
```
Comment (scopatz) [inline recipes/cdsdashboards/meta.yaml:22]: This should use `{{ PYTHON }}`

Comment (fcollonval) [reply recipes/cdsdashboards/meta.yaml:22]: Thanks for the review @scopatz 
The suggestion does not work. I tried it on https://github.com/conda-forge/staged-recipes/pull/11280.

In the mentioned PR, the work around was to use external script or I keep this syntax (as done in [jupyterhub-feedstock](https://github.com/conda-forge/jupyterhub-feedstock/blob/master/recipe/meta.yaml#L25)) as it is simpler than having two identical script (except for the os environment name).

Comment (scopatz) [reply recipes/cdsdashboards/meta.yaml:22]: I think this is picking up the wrong python then.  It should at least be `$PREFIX/bin/python`

Comment (fcollonval) [reply recipes/cdsdashboards/meta.yaml]: Thanks for the info. So I switched to external script to use $PYTHON/%PYTHON%.

File: recipes/cdsdashboards/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
```
Comment (scopatz) [inline recipes/cdsdashboards/meta.yaml:23]: If you rename these to `build.sh` and `bld.bat`, these two line are not needed, and therefore you don't need any selectors

Comment (fcollonval) [reply recipes/cdsdashboards/meta.yaml:23]: If I rename the script, it does not work as it tries to use them for the fake _meta_ package that comes with no host nor build environment.

Comment (scopatz) [reply recipes/cdsdashboards/meta.yaml:23]: Ahhh OK  - just keep the build_base.sh one then.  It is OK if this fails on windows/  Noarch packages are only built on linux

Comment (fcollonval) [reply recipes/cdsdashboards/meta.yaml]: Thanks for the tip. I made the correction.

File: recipes/cdsdashboards/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        - jupyterhub >=1.0.0
+        - alembic
+        - pluggy
+    test:
+      imports:
+        - cdsdashboards
+  - name: {{ name|lower }}-singleuser
+    build:
+      number: {{ build_number }}
+      noarch: generic
```
Comment (scopatz) [inline recipes/cdsdashboards/meta.yaml:40]: Is this noarch generic, not python? Just making sure!

Comment (fcollonval) [reply recipes/cdsdashboards/meta.yaml:40]: Sorry for the late answer. Yes it is generic as per a suggestion of @xhochy in [another recipe](https://github.com/conda-forge/staged-recipes/pull/11280/files/72f4a91638eb8846892cbc9bfe937e2b227768ae#diff-42f331d1ac0f9b69e3596208031be741)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**